src/database/queries/mentorExtension.js

```diff
@@ -182,12 +172,12 @@ module.exports = class MentorExtensionQueries {
 				additionalFilter = `${searchFilter.whereClause}`
 			}
 
-			let filterClause = filterConditions.length > 0 ? ` ${filterConditions.join(' AND ')}` : ''
+			const filterClause = filter.query.length > 0 ? `${filter.query}` : ''
```

Comment:
@nevil-mathew check filter.query exist or not then check the length

Reply:
Added safe chaining, but this response is returned from internal function, so this will always have `filter.query`.